src/workerd/api/util.c++

```diff
@@ -259,15 +257,6 @@ kj::String redactUrl(kj::StringPtr url) {
   return kj::String(redacted.releaseAsArray());
 }
 
-double dateNow() {
```

Comment:
Unrelated change?

Reply:
ah, nevermind, I see that things were shifted around a bit.

Reply:
yes, this change is required in order to avoid depending on io on this file, which eventually creates a circular dependency. the best way to avoid this by moving it to io-util.h